homeassistant/components/template/light.py

```diff
@@ -209,6 +221,8 @@ def __init__(
         self._white_value = None
         self._entities = entity_ids
         self._available = True
+        self._attribute_templates = attribute_templates
```

Comment:
This is probably not needed. You can simply access `attribute_templates` later on.